admin/src/tables/GeneratorProcessesTable.jsx

```diff
@@ -135,6 +155,7 @@ export default function GeneratorProcessesTable( { slug } ) {
 			cell: ( cell ) => <RowActionButtons
 				onDelete={ () => deleteRow( { cell, id: 'task_id' } ) }
 			>
+				<ActionButton cell={ cell } onClick={ ( val ) => updateRow( { changeField: 'status', newVal: val, cell } ) } />
```

Comment:
this button will not do anything, there is no update api for it
@yasha-dev1 pls add the api